assets/js/src/core/modules/element/dynamic-types/definitions/objects/data-related/components/localized-fields/localized-fields.tsx

```diff
@@ -22,7 +22,11 @@ export interface LocalizedFieldsProps extends AbstractObjectDataDefinition {
 }
 
 export const LocalizedFields = ({ children, noteditable, className }: LocalizedFieldsProps): React.JSX.Element => {
-  const { currentLanguage } = useLanguageSelection()
+  const { currentLanguage, setHasLocalizedFields } = useLanguageSelection()
+
+  useEffect(() => {
```

Comment:
I'm not sure if this can lead to performance problems. Maybe it's better to do a simple `if (!hasLocalizedFields) { setHasLocalizedFields(true) }`?